collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1236,7 +1238,7 @@ public void onClick(View v) {
                 }
 
                 if (mBackButton.isShown() && mNextButton.isShown()) {
-                    mBackButton.setEnabled(true);
+                    mBackButton.setEnabled(!firstQuestion);
```

Comment:
Okay, i tried that too. But that causes problems, if form has a EVENT_PROMPT_NEW_REPEAT event. Actually, stepToPreviousScreenEvent() and stepToNextScreenEvent() are not exactly opposite of each other.

Suppose there event A, next event B is EVENT_PROMPT_NEW_REPEAT, next event is C. We are on C currently. Now, calling stepToPreviousScreenEvent() returns event A and now calling stepToNextScreenEvent() returns event B. So, it causes problem that we get stuck at event C infinitely, pressing next button does not goes to next event.